python/src/scipp/plot/dispatch.py

```diff
@@ -14,8 +13,8 @@ def dispatch(scipp_obj_dict,
              mpl_line_params=None,
              **kwargs):
     """
-    Function to automatically dispatch the dict of scipp objects to the
-    appropriate plotting function depending on its dimensions.
+    Function to automatically dispatch the dict of DataArrays to the
```

Comment:
Can this now be a dict of DataArrays or variables?

Reply:
No, they are all DataArrays when they reach `dispatch`. I convert input Variables to DataArray by attaching fake coordinates in `plot.py`.